internal/controllers/gateway/httproute_controller.go

```diff
@@ -644,106 +630,6 @@ func (r *HTTPRouteReconciler) getHTTPRouteRuleReason(ctx context.Context, httpRo
 	return gatewayv1beta1.RouteReasonResolvedRefs, nil
 }
 
-// ensureParentsAcceptedCondition sets the "Accepted" condition of HTTPRoute status.
-// returns a non-nil error if updating status failed,
-// and returns true in the first return value if status changed.
-func (r *HTTPRouteReconciler) ensureParentsAcceptedCondition(
```

Comment:
I'm just wondering why we don't need this anymore 🤔 . We're setting this elsewhere or for some reason we don't need to set the parent status?